From 7eb28814e97d0f2b155de96d6b21fa27d80dbee7 Mon Sep 17 00:00:00 2001
From: Felipe Angel <fangel@hugeinc.com>
Date: Sat, 11 Jun 2016 23:49:54 -0500
Subject: [PATCH] adding silk product

---
 full_victara.mk | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/full_victara.mk b/full_victara.mk
index cb24c1a..443f712 100644
--- a/full_victara.mk
+++ b/full_victara.mk
@@ -14,8 +14,10 @@
 # limitations under the License.
 #
 
+$(call inherit-product, product/$(SILK_PRODUCT)/$(SILK_PRODUCT).mk)
+
 # Inherit from those products. Most specific first.
-$(call inherit-product, $(SRC_TARGET_DIR)/product/full_base_telephony.mk)
+$(call inherit-product, vendor/motorola/victara/victara-vendor-blobs.mk)
 
 # Inherit from victara device
 $(call inherit-product, device/motorola/victara/device.mk)
-- 
2.5.4 (Apple Git-61)

