From 61646f06cf30cbc6233c99e10de1e0bcaca9999b Mon Sep 17 00:00:00 2001
From: angelpipe <angelpipe@gmail.com>
Date: Mon, 13 Jun 2016 08:25:40 -0500
Subject: [PATCH] Add silk product

---
 full_victara.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/full_victara.mk b/full_victara.mk
index b3604a2..c6a0925 100644
--- a/full_victara.mk
+++ b/full_victara.mk
@@ -17,6 +17,9 @@
 # Get the long list of APNs
 PRODUCT_COPY_FILES := device/sample/etc/apns-full-conf.xml:system/etc/apns-conf.xml
 
+# Silk product
+$(call inherit-product, product/$(SILK_PRODUCT)/$(SILK_PRODUCT).mk)
+
 # Inherit from those products. Most specific first.
 $(call inherit-product, $(SRC_TARGET_DIR)/product/aosp_base_telephony.mk)
 
-- 
2.8.4

